packages/amazonq/.changes/next-release/Bug Fix-34e877a1-d2ca-4c93-9dee-5dd2019769fe.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Enabled users to view diffs by clicking files in the file tree, aligning the behavior with the 'View Diff' button."
```

is this for a particular command? if so, it helps to mention it. something like:

```suggestion
	"description": "`/test`: view diffs by clicking files in the file tree, aligning the behavior with the 'View Diff' button."
```